src/dataviz/adapters/chartjs.js

```diff
@@ -143,6 +148,12 @@ module.exports = function(){
     }
   }
 
+  function _isEmptyOutput(dataset) {
```

Comment:
I'm assuming the `flatten` module doesn't do what's required to flatten this data structure?